src/err.rs

```diff
@@ -40,6 +42,40 @@ pub struct NlmsghdrErr<T> {
     pub nl_seq: u32,
     /// ID of the netlink destination for requests and source for responses
     pub nl_pid: u32,
+    nl_payload: Buffer,
```

Comment:
Maybe you can add a convenient doc comment, like: `Optionally the payload of the original message that caused the error.`?